libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -407,7 +407,7 @@ class MapboxTripSession(
 
     private suspend fun getNavigatorStatus(date: Date): TripStatus =
         withContext(ioJobController.scope.coroutineContext) {
-            date.time = date.time + navigatorPollingDelay
+            date.time = date.time + navigatorPredictionIntervalLag
```

Comment:
💯 on renaming navigatorPollingDelay, hard to know what that means
interval lag is honestly a little confusing to me too.

Some more ideas, it's a prediction time, time offset, or time to look ahead
- [ ] locationPredictionMillis
- [ ] predictLocationMillis
- [ ] predictAheadMillis
- [ ] navigatorPredictionMillis

Reply:
think navigatorPredictionMillis is my favorite

Reply:
What do you think of `predictionLookAheadMillis`?

Thanks for the suggestions - I've brought the naming from the legacy 😅 https://github.com/mapbox/mapbox-navigation-android/blob/3065dbfee452db030294e083f9d8a2e612bd35ae/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationConstants.kt#L112-L122 https://github.com/mapbox/mapbox-navigation-android/blob/3065dbfee452db030294e083f9d8a2e612bd35ae/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationOptions.kt#L24

Reply:
Ah that's where interval lag came from. Yeah i like verbose, but not navigationLocationEngineIntervalLagInMilliseconds 😄 

`predictionLookAheadMillis` is good too

Reply:
possibly redundant because prediction and lookAhead are the same thing

Reply:
`navigatorPredictionMillis` then 🚀